src/azure-cli/azure/cli/command_modules/resource/_help.py

```diff
@@ -388,6 +395,8 @@
     short-summary: The path to the template file.
   - name: --template-uri -u
     short-summary: The URI to the template file.
+  - name: --template-spec -s
```

Comment:
Actually it is not a good experience to write parameter description here with so many duplicated code.
Prefer to add it in _params.py with only one representation

Reply:
Done.

Reply:
@detienne20 Could you please delete all the extra help?  `- name: --template-spec -s`
I saw you have added this help here:
 `deployment_template_spec_type = CLIArgumentType(options_list=['--template-spec', '-s'], is_preview=True, min_api='2019-06-01', help="The template spec resource id.")` 
So the above ones are repeated

Reply:
This is consistent with the template_file and template_uri parameters. in _param.pys and _help.py: c.argument('template_file', arg_type=deployment_template_file_type)
        c.argument('template_uri', arg_type=deployment_template_uri_type),     deployment_template_file_type = CLIArgumentType(options_list=['--template-file', '-f'], completer=FilesCompleter(), type=file_type,
                                                    help="a template file path in the file system"), deployment_template_uri_type = CLIArgumentType(options_list=['--template-uri', '-u'], help='a uri to a remote template file')